src/Packagist/WebBundle/Entity/PackageRepository.php

```diff
@@ -279,7 +279,7 @@ public function getFilteredQueryBuilder(array $filters = array())
             $qb->leftJoin('v.tags', 't');
         }
 
-        $qb->orderBy('p.id', 'DESC');
```

Comment:
But if I write:

```php
$qb->orderBy('id', 'DESC');
$qb->addOrderBy('p.abandoned');
```

And want to change the order by name on user controller, it will erases the `p.abandoned` order.

Am I right? In this case, what would be the solution?

And BTW, the `orderBy` is finally set on each controller, so what is the problem?